!SystemError: with a message
	in System_error3.foo() [System_error3.v3 @ 7:29]
	in System_error3.main() [System_error3.v3 @ 3:20]

